src/plugins/ui/evaluation_chart.ts

```diff
@@ -231,6 +231,10 @@ export class EvaluationChartPlugin extends UIPlugin {
       }
     }
     for (const [dsIndex, ds] of Object.entries(definition.dataSets)) {
+      const data = ds.dataRange ? this.getData(ds, definition.sheetId) : [];
+      if (!data.filter(isDefined).length) {
```

Comment:
```suggestion
      if (!data.every((cell) => cell === undefined)) {
```
nitpick. This won't reallocate a new array like `filter`

Also it seems strange to me that it works at all. Don't evaluated cells return empty strings for empty cells ? Why is checking that the value is undefined works then ? Or wasn't that the case in 15.0 ?

Reply:
Maybe it was only the case in 15.0, but here when we have empty cell in the dataset we get the "undefined" value. But I'll check in the foward-port if it still works for the next versions :)